simpeg_drivers-assets/uijson/direct_current_forward_2d.ui.json

```diff
@@ -20,7 +20,9 @@
             "{48f5054a-1c5c-4ca4-9048-80f36dc60a06}",
             "{b020a277-90e2-4cd7-84d6-612ee3f25051}"
         ],
-        "value": ""
+        "value": "",
+        "optional": true,
```

Comment:
topo is optional?  what will the active default to if not set?

Reply:
The active_model becomes non-optional if topo is off, as desired.

![image](https://github.com/user-attachments/assets/83456633-7253-4efd-9835-a5f56cdfa39c)

Reply:
ok, that does make sense.. just couldn't picture it.  Now I wonder if the active model should go into the Mesh and Models section to avoid scrolling down to select mesh, then back up to pick the active model.